lib/FunctionIsolation/RemoveExceptionalCalls.cpp

```diff
@@ -40,11 +41,16 @@ bool RemoveExceptionalCalls::runOnModule(llvm::Module &M) {
     // Split containing basic block
     BB->splitBasicBlock(Call);
 
+    // TODO: investigate whether it's safe to use this after the termination
+    //       was dropped.
+    DILocation *DI = BB->getTerminator()->getDebugLoc();
+
     // Drop terminator of the old basic block
     eraseFromParent(BB->getTerminator());
 
     // Terminate with an unreachable
-    new UnreachableInst(C, BB);
+    auto Unreachable = new UnreachableInst(C, BB);
+    Unreachable->setDebugLoc(DI);
```

Comment:
It shouldn't be unsafe to use `DI`, but if you want to be sure just do:
```cpp
   DILocation *DI = BB->getTerminator()->getDebugLoc();
   auto *OldTerminator = BB->getTerminator();
   auto Unreachable = new UnreachableInst(C, BB);
   Unreachable->setDebugLoc(DI);
   eraseFromParent(OldTerminator);
   ```

Reply:
Ah, this was a todo for myself 😅.

This should have never been pushed, my bad.